.dev/scripts/generate_changelog.py

```diff
@@ -149,7 +148,7 @@ def make_pr_bullet_point(pr, prefix=None):
     if prefix in ("modules", "categories"):
         tags = "".join(["[" + k + "]" for k in pr[prefix]])
     if tags:
-        tags = "**" + tags + "**"
+        tags = "**" + tags + "** "
 
     return f"* {tags}{pr['title']} [{ref}]"
```

Comment:
```suggestion
        tags = "**" + tags + "** "

    return f"* {tags} {pr['title']} [{ref}]"
```

Reply:
Why? If `tags` is empty we'll get a double-space.

Reply:
I thought this was a formatting change, but it's not strictly a formatting change.